src/state/musicCollection.ts

```diff
@@ -1,76 +1,44 @@
-import { type Repo, isValidAutomergeUrl } from "@automerge/automerge-repo";
-import { useDocument, useRepo } from "@automerge/automerge-repo-react-hooks";
+import { useDocument } from "@automerge/automerge-repo-react-hooks";
 import { useState } from "react";
-
-function getOrCreateDocument<V>(
-	repo: Repo,
-	name: string,
-	init: (d: V) => void = () => {},
-) {
-	const documentURL = localStorage.getItem(name);
-
-	if (isValidAutomergeUrl(documentURL)) {
-		return documentURL;
-	}
-
-	const handle = repo.create<V>();
-	handle.change(init);
-
-	localStorage.setItem(name, handle.url);
-
-	return handle.url;
-}
-
-export type MusicCollectionItem = {
-	fileName: string;
-	title: string;
-};
-
-export type MusicCollectionDocument = { collection: MusicCollectionItem[] };
-
-export function getDocumentsURLs(repo: Repo) {
-	return {
-		musicCollection: getOrCreateDocument<MusicCollectionDocument>(
-			repo,
-			"musicCollection",
-			(d) => {
-				d.collection = [];
-			},
-		),
-	};
-}
-
-export type Documents = ReturnType<typeof getDocumentsURLs>;
+import type { MusicCollection, MusicItem } from "./schema";
+import { useUserDocuments } from "./user";
+import { copyToPrivateFileSystem } from "@/lib/filesystem";
 
 export function useMusicCollection() {
-	const repo = useRepo();
-	const documentUrl = useState(() => getDocumentsURLs(repo).musicCollection)[0];
-	const [doc, change] = useDocument<MusicCollectionDocument>(documentUrl);
+	const { collectionsUrls } = useUserDocuments();
```

Comment:
Now we store in a document all the user music collections.

This way in the future we can show the collections list